website/docs/advanced/configuring-dns.md

```diff
@@ -16,7 +16,7 @@ In this guide, we will bind the document issuer's identity to a valid domain nam
 
 To bind the domain name to the issuer's identity, you must be able to change the DNS record of the domain name.
 
-## Inserting the DNS Record
+## Inserting the DNS Record for Ethereum Smart contracts
```

Comment:
Smart contracts seem a bit vague, hmmm....
maybe just say Document Store and Token Registry

Reply:
After discussion (on ehour) we decided to keep it like this